test/features/events/response.test.ts

```diff
@@ -288,3 +288,27 @@ it('fetch: emits the "response" event upon the original response', async () => {
 
   expect(isMockedResponse).toBe(false)
 })
+
+it('supports reading the request and response bodies in the "response" listener', async () => {
+  const requestCallback = vi.fn()
+  const responseCallback = vi.fn()
+  const responseListener = vi.fn<HttpRequestEventMap['response']>(
+    async ({ request, response }) => {
```

Comment:
All the existing tests are already reading the same request/response instances from the `response` event callback, so I've added this new case when that's done within the callback itself and sequentially (similar to the reported issue).